ostd/src/arch/riscv/boot/boot.S

```diff
@@ -46,7 +47,7 @@ _start:
     # Check if the write to satp succeeds. If not, try Sv39.
     # Reference: <https://riscv.github.io/riscv-isa-manual/snapshot/privileged/#satp>.
     csrr   t1, satp
-    beq    t0, t1, flush_tlb_bsp
+    beq    t0, t1, bsp_flush_tlb
```

Comment:
Should we rename `boot.S` to `bsp_boot.S`? This is more consistent with the x86 version. Also, given that the symbols are prefixed with `bsp_`, it should make more sense.